src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -3067,6 +3067,51 @@ public void ValidateGraphQLSchemaForCircularReference(string schema)
             Assert.AreEqual(DataApiBuilderException.SubStatusCodes.ErrorInInitialization, exception.SubStatusCode);
         }
 
+        [TestMethod, TestCategory(TestCategory.COSMOSDBNOSQL)]
```

Comment:
nit: helpful summary would give context:
GraphQL Schema types defined -> `Character` and  `Planet`
DAB runtime config entities defined -> `Planet` (Not defined: `Character`)
Mismatch of entities and types between provided GraphQL schema file and DAB config results in actionable error message.

Reply:
I thought function name is quite clear and test is very straight forward...but yes, i will put the comment also, as you said